tests/ask.bats

```diff
@@ -46,12 +48,12 @@ setup() {
 @test "ask env: should print messages if parameters need to set for different component discriminators" {
   run ask env "ARGO_HOST" -m "parameter ingress.hosts test1" +empty -d "$DOTENV_FILE" -ask-env --non-interactive
   assert_success
-  assert_output --partial "* Setting parameter ingress.hosts test1, ARGO"
+  assert_output --partial "* Setting parameter ingress.hosts test1"
```

Comment:
@NellyHargeliya please rename implementation to better naming `* Setting parameter `ingress.hosts (component: test1)`